src/query_ctx.c

```diff
@@ -105,12 +113,14 @@ Graph *QueryCtx_GetGraph(void) {
 
 RedisModuleCtx *QueryCtx_GetRedisModuleCtx(void) {
 	QueryCtx *ctx = _QueryCtx_GetCtx();
-	return ctx->global_exec_ctx.redis_ctx;
+	if(ctx) return ctx->global_exec_ctx.redis_ctx;
+	return NULL;
 }
 
 ResultSet *QueryCtx_GetResultSet(void) {
 	QueryCtx *ctx = _QueryCtx_GetCtx();
-	return ctx->internal_exec_ctx.result_set;
+	if(ctx) return ctx->internal_exec_ctx.result_set;
```

Comment:
this is a general comment, for the getter functions, should we call `_QueryCtx_GetCtx` and assert `ctx` exists?